addons/stock/models/res_config_settings.py

```diff
@@ -58,6 +58,8 @@ def _onchange_group_stock_multi_locations(self):
     def _onchange_group_stock_production_lot(self):
         if not self.group_stock_production_lot:
             self.group_lot_on_delivery_slip = False
+            if self.env['product.product'].search([('tracking', '!=', 'none')], limit=1):
```

Comment:
```suggestion
            if self.env['product.product'].search_count([('tracking', '!=', 'none')], limit=1):
```
As you don't need the result of the search, only the count

Reply:
done :+1: